indigo/docs/quickstart/hello-indigo.md

```diff
@@ -377,7 +377,7 @@ def updateModel(
     context: FrameContext[Unit],
     model: Model
 ): GlobalEvent => Outcome[Model] = {
-  case MouseEvent.Click(clickPoint) =>
+  case MouseEvent.Click(clickPoint, _, _, _, _, _, _, _) =>
```

Comment:
Hmmm yes that is a lot. It's probably the right thing to do but I'm going to sleep on it.

Reply:
I've had a thought about this. I thin (in this instance at least) `case e: MouseEvent.Click` would be a better way of expressing this. I've updated the examples to favour that approach